code/src/store/api/cartApi.ts

```diff
@@ -0,0 +1,26 @@
+import { rtkQueryTags } from "@/constants/api-tags";
+import { httpMethods } from "@/constants/methods";
+import { URLS } from "@/constants/requests";
+import { appApi } from "@/store/api/appApi";
+import { CartManagementParams } from "@/types/cart.types";
+
+const cartApi = appApi.injectEndpoints({
+  endpoints: (build) => ({
+    addToCart: build.mutation<void, CartManagementParams>({
+      query: (args) => ({
+        url: URLS.cart.post(args),
+        method: httpMethods.post
+      }),
+      invalidatesTags: [rtkQueryTags.CART]
```

Comment:
Do we need to use tags here? As far as I know, tags are commonly used on `GET` queries. If I am mistaken, please correct me

Reply:
Tags are used to make automatic refetch (`GET`) on some action (`POST`, `PATCH `or `DELETE`). Now we do not have `GET` request, but soon we will have it. `invalidatesTags` accepts tags, which should be refetched after mutation. So we will provide the same tag to our query for cart and it will be automatically refetched on created in my PR requests.

Reply:
Interesting. Then we should also add tags to existing rtk endpoints